### PR TITLE
chore: add changelog entries for #6181

### PR DESCRIPTION
**Pull request by thomaseizinger**
Add either new changelog entries for next version or modify existing changelog entries to point out #6181.

### CODE REVIEW

File: website/src/components/Changelog/Android.tsx [hunk 1]
```diff
@@ -15,6 +15,9 @@ export default function Android() {
           <ChangeItem pull="6107">
             Adds the ability to mark Resources as favorites.
           </ChangeItem>
+          <ChangeItem pull="6181">
+            Improves reliability of DNS resolution of non-resources.
+          </ChangeItem>
```
Comment (jamilbk) [inline website/src/components/Changelog/Android.tsx:20]:
```suggestion
          {/* <ChangeItem pull="6181">
            Improves reliability of DNS resolution of non-resources.
          </ChangeItem> */}
```

File: website/src/components/Changelog/GUI.tsx
Comment (jamilbk) [inline website/src/components/Changelog/GUI.tsx:1]: Just needs the mention about the fix because the panic happens in 1.1.10 GUI client

**Review by jamilbk — APPROVED** [review]

File: website/src/components/Changelog/Android.tsx (hunk 1 above)
Comment (thomaseizinger) [reply website/src/components/Changelog/Android.tsx:20]: The entire `Entry` is commented out already! :)

File: website/src/components/Changelog/GUI.tsx
Comment (thomaseizinger) [reply website/src/components/Changelog/GUI.tsx:1]: See line 24-26.

**Comment by jamilbk** [comment]
Merging so I can push these out today

**Pull request merged**